shell/common/engine.h

```diff
@@ -716,6 +716,20 @@ class Engine final : public RuntimeDelegate, PointerDataDispatcher::Delegate {
   // |PointerDataDispatcher::Delegate|
   void ScheduleSecondaryVsyncCallback(fml::closure callback) override;
 
+  //----------------------------------------------------------------------------
+  /// @brief      Get the last Entrypoint that was used in the RunConfiguration
+  ///             when |Engine::Run| was called.
+  ///
+  const std::string& GetEntrypoint() const { return entry_point_; }
+
+  //----------------------------------------------------------------------------
+  /// @brief      Get the last Entrypoint Library that was used in the
+  ///             RunConfiguration when |Engine::Run| was called.
+  ///
+  const std::string& GetEntrypointLibrary() const {
```

Comment:
`GetLastEntryPointLibrary`. Also, please OOL this accessor.

Reply:
done